### PR TITLE
Student team update

### PR DESCRIPTION
**Pull request by Krishnansh5**

### CODE REVIEW

**Comment by abhishekshree** [comment]
Images are not rendering.

File: pages/student-team.tsx [hunk 1]
```diff
@@ -10,159 +10,117 @@ import { media } from 'utils/media';
 const TeamMembers = [
   {
     name: 'Gyanendra Kumar',
-    image: '/testimonials/iitk-logo.svg',
-    phone: '(+91) 78965 78965',
-    mail: 'xyz@iitk.ac.in',
-    linkedin: 'https://www.linkedin.com/in/abc'
+    image: '/testimonials/Gyanendra.jpg',
```
Comment (abhishekshree) [inline pages/student-team.tsx:13]: check `.jpg` everywhere, some images have `.JPG`

**Review by abhishekshree — CHANGES_REQUESTED** [review]

File: pages/student-team.tsx (hunk 1 above)
Comment (1-Harshit) [reply pages/student-team.tsx:13]: rename file names not literals

**Comment by 1-Harshit** [comment]
Why did you delete office staff images?

**Comment by 1-Harshit** [comment]
Crop image 1:1

**Pull request merged**